pandas/tests/indexes/test_base.py

```diff
@@ -1565,6 +1565,79 @@ def test_drop_tuple(self, values, to_drop):
         for drop_me in to_drop[1], [to_drop[1]]:
             pytest.raises(KeyError, removed.drop, drop_me)
 
+    def test_drop_unique_and_non_unique_index(self):
```

Comment:
any way to parameterize some of this (e.g. you can break into 2 tests if that makes it easier)

Reply:
I told you to move them here, but on 2nd thought, there they belong in test_alter_index (where the tests were originally). sorry about that.

test_base is for generically testing Index/Series types and not doing that here so kind of superfluous to put the tests here.